File: Groupe2/TP1/src/operateurs.c
// operateurs.c

#include <stdio.h>

int main() {
    int a = 16;
    int b = 3;
    float division_result;  // Stocker le résultat de la division

    // Opérations arithmétiques
    int addition = a + b;
    int soustraction = a - b;
    int multiplication = a * b;
    division_result = (float)a / b;  // Pour obtenir un résultat flottant
    int modulo = a % b;

    // Opérations logiques
    int egalite = (a == b);
    int superieur = (a > b);

    // Résultats
    printf("Addition : %d\n", addition);
    printf("Soustraction : %d\n", soustraction);
    printf("Multiplication : %d\n", multiplication);
    printf("Division : %.2f\n", division_result);
    printf("Modulo : %d\n", modulo);
    printf("a est egal a b : %s\n", egalite ? "Vrai" : "Faux");
    printf("a est superieur a b : %s\n", superieur ? "Vrai" : "Faux");

    return 0;
}

File: Groupe2/TP1/src/operateurs.py
# operateurs.py

# Déclaration des variables
a = 16
b = 3

# Opérations arithmétiques
addition = a + b
soustraction = a - b
multiplication = a * b
division = a / b  # Retourne toujours un float en python
modulo = a % b

# Opérations logiques
egalite = (a == b)
superieur = (a > b)

# Résultats
print(f"Addition : {addition}")
print(f"Soustraction : {soustraction}")
print(f"Multiplication : {multiplication}")
print(f"Division : {division:.2f}")  # Affiche 2 décimales
print(f"Modulo : {modulo}")
print(f"a est égal à b : {'Vrai' if egalite else 'Faux'}")
print(f"a est supérieur à b : {'Vrai' if superieur else 'Faux'}")

#Ici ont peut utiliser des "é" ou "à" sans problème d'affichage